sncosmo/models.py

```diff
@@ -1128,8 +1129,8 @@ def _bandflux_rvar_single(self, band, phase):
         m0 = self._model['M0'](phase, wave)
         m1 = self._model['M1'](phase, wave)
         tmp = trans * wave
-        f0 = np.sum(m0 * tmp, axis=1) * dwave / HC_ERG_AA
-        m1int = np.sum(m1 * tmp, axis=1) * dwave / HC_ERG_AA
+        f0 = np.sum(m0 * tmp, axis=1)/tmp.sum()
+        m1int = np.sum(m1 * tmp, axis=1)/tmp.sum()
```

Comment:
This seems like a big change from SALT2. Can you explain the logic and/or add a comment here?

Reply:
thanks Kyle, will do - this evaluates the average M0 + x1*M1 for a given bandpass.  The variance, "v" is the uncertainty in M0 + x1*M1 (in SALT2 it's a fractional error but here it's a true error).  So then to return the relative variance we divide v/ftot**2